rmw_fastrtps_shared_cpp/src/rmw_take.cpp

```diff
@@ -131,6 +182,40 @@ __rmw_take(
   return _take(identifier, subscription, ros_message, taken, nullptr, allocation);
 }
 
+rmw_ret_t
+__rmw_take_sequence(
+  const char * identifier,
+  const rmw_subscription_t * subscription,
+  size_t count,
+  rmw_message_sequence_t * message_sequence,
+  rmw_message_info_sequence_t * message_info_sequence,
+  size_t * taken,
+  rmw_subscription_allocation_t * allocation)
+{
+  RCUTILS_CHECK_FOR_NULL_WITH_MSG(
+    subscription, "subscription pointer is null", return RMW_RET_ERROR);
+  RCUTILS_CHECK_FOR_NULL_WITH_MSG(
+    message_sequence, "message_sequence pointer is null", return RMW_RET_ERROR);
+  RCUTILS_CHECK_FOR_NULL_WITH_MSG(
+    message_info_sequence, "message_info_sequence pointer is null", return RMW_RET_ERROR);
+  RCUTILS_CHECK_FOR_NULL_WITH_MSG(
+    taken, "size_t flag for count is null", return RMW_RET_ERROR);
```

Comment:
I think the above error codes should be changed to `RMW_RET_INVALID_ARGUMENT`, right?

Reply:
c493ccc